R/kiss-rule.R

```diff
@@ -0,0 +1,29 @@
+
+#' @export
+KissRule.Event <- function(negate, eventId, frequencyValue, frequencyOccurance, comparisonMode) {
+  structure(list(
+    type = "event",
+    negate = negate,
+    event = eventId,
+    frequencyValue = frequencyValue,
+    frequencyOccurance = frequencyOccurance,
+    comparisonMode = comparisonMode),
+    class = c("KissRule.Event", "KissRule"))
+}
+
+#' @export
+KissRule.Property <- function(negate, propertyId, comparisonMode) {
```

Comment:
Would be nice to be able to pass in an actual kiss-property as well - and have that autofill the propertyId